common-codegen/src/test/java/io.vrap.rmf.codegen.common/CodeGeneratorTest.java

```diff
@@ -47,11 +43,11 @@ public void init() {
         }
 
         final ClassLoader classLoader = getClass().getClassLoader();
-        final Path inputPath = null;
+        final Path inputPath = Paths.get("/Users/mkoester/Development/commercetools-importer/api-spec/api.raml");;
```

Comment:
The tests here contains local paths maybe we should replace them by relative paths, WDYT?

Reply:
Good catch, I even didn't want to push these changes 😉 But I like your idea and will change it 😄